nvflare/private/fed/client/communicator.py

```diff
@@ -194,14 +195,13 @@ def pull_task(self, servers, project_name, token, ssid, fl_ctx: FLContext):
         return_code = task.get_header(MessageHeaderKey.RETURN_CODE)
 
         if return_code == ReturnCode.OK:
-            size = len(task.payload)
-            task.payload = task.payload
+            size = task.get_header(MessageHeaderKey.PAYLOAD_LEN)
             task_name = task.payload.get_header(ServerCommandKey.TASK_NAME)
             fl_ctx.set_prop(FLContextKey.SSID, ssid, sticky=False)
             if task_name not in [SpecialTaskName.END_RUN, SpecialTaskName.TRY_AGAIN]:
                 self.logger.info(
-                    f"Received from {project_name} server "
-                    f" ({size} Bytes). getTask: {task_name} time: {end_time - start_time} seconds"
+                    f"Received from {project_name} server. getTask: {task_name} size:{format_size(size)} "
```

Comment:
Please add a space btw "size:" and "{format_size(size)".

Reply:
Done.